### ISSUE TITLE
dev-libs/granite-5.2.1: cmake-utils.eclass could not be found by inherit()

### ISSUE THREAD

**Issue by triffid** [Gentoo-zh/gentoo-zh#2402]
```
* ERROR: dev-libs/granite-5.2.1::gentoo-zh failed (depend phase):
*   cmake-utils.eclass could not be found by inherit()
* 
* Call stack:
*              ebuild.sh, line 618:  Called source '/var/db/repos/gentoo-zh/dev-libs/granite/granite-5.2.1.ebuild'
*   granite-5.2.1.ebuild, line   8:  Called inherit 'cmake-utils' 'gnome2-utils' 'vala'
*              ebuild.sh, line 264:  Called die
* The specific snippet of code:
*              [[ -z ${location} ]] && die "${1}.eclass could not be found by inherit()"
```

https://github.com/microcai/gentoo-zh/blob/2afaf8dc2432a83b7ebb593613484091854a9c1d/dev-libs/granite/granite-5.2.1.ebuild#L8 vs https://github.com/microcai/gentoo-zh/tree/2afaf8dc2432a83b7ebb593613484091854a9c1d/eclass vs https://gitweb.gentoo.org/repo/sync/gentoo.git/tree/eclass

Perhaps it was renamed or removed?

**Comment by triffid** [comment]
Just synced and this seems to be solved (via 1dc829d I guess), but now it's saying `cmake: EAPI=6 is not supported`

**Comment by triffid** [comment]
Full error after 1dc829d:

```
* ERROR: dev-libs/granite-5.2.1::gentoo-zh failed (depend phase):
*   cmake: EAPI=6 is not supported
*
* Call stack:
*              ebuild.sh, line 618:  Called source '/var/db/repos/gentoo-zh/dev-libs/granite/granite-5.2.1.ebuild'
*   granite-5.2.1.ebuild, line   8:  Called inherit 'cmake' 'gnome2-utils' 'vala'
*              ebuild.sh, line 298:  Called __qa_source '/var/db/repos/gentoo/eclass/cmake.eclass'                                                                                                                                                          a *              ebuild.sh, line 114:  Called source '/var/db/repos/gentoo/eclass/cmake.eclass'
*           cmake.eclass, line  23:  Called die
* The specific snippet of code:
*      *) die "${ECLASS}: EAPI=${EAPI:-0} is not supported" ;;
```
